java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/ArrowDatabaseMetadataTest.java

```diff
@@ -269,6 +269,18 @@
   private static Connection connection;
 
   static {
+    UserPasswordAuthentication authentication =
```

Comment:
I think we can keep a `FlightServerTestRule.createNewTestRule` like before that uses the builder internally.

Most test files changed to this pattern of building the FlightServerTestRule manually, but we can leave them unmodified

Reply:
Fixed